### PR TITLE
Badge

### PR DESCRIPTION
**Pull request by arpit01923**
Badge component is created

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    box-sizing: border-box;
+}
+
+:root{
+    --avatar-border-radius: 50%;
+
+    --badge-online: green;
+    --badge-offline: thistle;
+    --badge-dnd: red;
+
```
Comment (Aditi0505) [inline css/style.css]: Suggestion : Can remove extra spaces.

**Review by Aditi0505 — APPROVED** [review]

**Review by aniketsingh13 — APPROVED** [review]
good to go..

File: css/components/Badge.css [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+.badge .fas{
+   font-size: 2rem; 
+}
+
+.badge .number{
+    background-color: var(--active);
+    color: var(--white-section);
+    border-radius: var(--avatar-border-radius);
+    position: absolute;
+}
```
Comment (bajajcodes) [inline css/components/Badge.css]: Add new line here

File: css/style.css (hunk 1 above)
Comment (arpit01923) [reply css/style.css]: conflict resolved

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+}
+
+body{
+    font-size: 62.5%;
+}
+
+.sm{
+    width: 3rem;
+    height: 3rem;
+}
```
Comment (bajajcodes) [inline css/style.css]: Add new line also here.

File: css/style.css (hunk 1 above)
Comment (arpit01923) [reply css/style.css]: and Thanks for the suggestion

**Review by bajajcodes — COMMENTED** [review]
- Add new line at the end of file.
- Show output either via image or video, code is not enough for visualizing what your trying to communicate.

File: css/components/Badge.css (hunk 2 above)
Comment (arpit01923) [reply css/components/Badge.css]: where after border-radius

Comment (arpit01923) [reply css/components/Badge.css]: or at the end

Comment (arpit01923) [reply css/components/Badge.css]: conflict resolved

File: css/style.css (hunk 3 above)
Comment (arpit01923) [reply css/style.css]: conflict resolved

Comment (arpit01923) [reply css/style.css]: And thanks a lot for the suggestions

**Comment by arpit01923** [comment]
image

**Pull request merged**